### PR TITLE
Better install instructions

### PR DESCRIPTION
**Pull request by shimwell**
## Proposed changes

I have been getting a few questions on how to install the parmak so I thought I would improve the docs in this area and include instructions for the Conda part.

## Types of changes

What types of changes does your code introduce to the Paramak?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [x] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Pep8 applied
- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,19 +71,27 @@ attribution by citing our [Publication](https://f1000research.com/articles/10-27
 
 ## System Installation
 
-To install the Paramak you need to have 
-[Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/), 
-[Cadquery 2](https://cadquery.readthedocs.io/en/latest/installation.html) and 
-[Pip](https://anaconda.org/anaconda/pip). If you have these three dependencies 
-already then you can install the Paramak using Pip:
+To install the Paramak you need to have Conda or MinConda installed,
```
Comment (billingsley-john) [inline README.md]:
```suggestion
To install the Paramak you need to have Conda or MiniConda installed,
```

**Comment by shimwell** [comment]
Thanks for the correction @billingsley-john

File: README.md [hunk 2]
```diff
@@ -71,19 +71,27 @@ attribution by citing our [Publication](https://f1000research.com/articles/10-27
...
-already then you can install the Paramak using Pip:
+To install the Paramak you need to have Conda or MiniConda installed,
+[Conda instructions here](https://docs.conda.io/projects/conda/en/latest/user-guide/install/),
+ [Miniconda instructions here.](https://docs.conda.io/en/latest/miniconda.html)
 
+Once you have Conda or MiniConda installed you can use this command to create a
+new conda enviroment, install CadQuery and activate the enviroment.
+
+```bash
+conda create -n paramak_env -c conda-forge -c cadquery python=3.8 cadquery=2
```
Comment (RemDelaporteMathurin) [inline README.md]:
```suggestion
conda create -n paramak_env -c conda-forge -c cadquery python=3.8 cadquery=2.1
```

we're using 2.1 now

File: docs/source/index.rst [hunk 3]
```diff
@@ -59,29 +59,38 @@ The Paramak source code is distributed with a permissive open-source license
...
+  
+Once you have Conda or MiniConda installed then CadQuery can be installed
+into a new enviroment and that environment can be activated using Anaconda or Miniconda. 
 
-Once you have activated a conda environment, Cadquery 2.0 can be installed
+Cadquery 2 can be installed in a Conda environment via conda-forge.
+
+.. code-block:: python
+
+   conda create -n paramak_env -c conda-forge -c cadquery python=3.8 cadquery=2
```
Comment (RemDelaporteMathurin) [inline docs/source/index.rst]:
```suggestion
   conda create -n paramak_env -c conda-forge -c cadquery python=3.8 cadquery=2.1
```

File: README.md (hunk 2 above)
Comment (shimwell) [reply README.md]: I thought cadquery=2 just took the newest release which is 2.1

**Pull request merged**